index.rst

```diff
@@ -1,59 +1,367 @@
-..
-  Technote content.
 
-  See https://developer.lsst.io/restructuredtext/style.html
-  for a guide to reStructuredText writing.
+:tocdepth: 1
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
 
-  Use the following syntax for sections:
+.. sectnum::
 
-  Sections
-  ========
+.. TODO: Delete the note below before merging new content to the master branch.
 
-  and
+.. note::
 
-  Subsections
-  -----------
+   **This technote represents a policy proposal that has not yet been adopted.**
 
-  and
+While the Gen3 Butler provides some intrinsic structure to its data repositories, considerably more is left to convention (often encoded in higher-level packages, like obs_base).  This document will be - at least at first - a proposal for how to organize data repositories in detail, focusing on collection naming conventions, filesystem locations, and developer workflows.  The immediate focus will be the environment at NCSA, but it is hoped that much of this will hold for the IDF and USDF as well.
 
-  Subsubsections
-  ^^^^^^^^^^^^^^
+After consultation with other stakeholders and ultimately RFC, at least some of the content here should probably be moved to the DM Developer Guide.
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
+Collections
```

Comment:
I hope we'll get to that level of documentation eventually, and I certainly understand that it's hard to read a policy proposal like this without the background needed to make sense of it (especially when documentation for that background doesn't make sense), but I just don't plan to evolve this into a user guide or Gen3 intoduction.  The daf_butler package docs and various scattered live-tutorial materials (which need to be integrated into the pipeline docs, of course) are already better for that, even if "better" != "good".

On the scope question: that was exactly what I intended to cover in the abstract (i.e. the paragraph above this heading).  Having gotten to the end, is there something specific I should have included in the abstract that would have helped, or some other reordering of this section with the others that would have worked better?  I did terminology right after the abstract specifically because I was worried about leading with undefined jargon, and felt it was appropriate to cover a little bit of that instead of just linking back to daf_butler docs.  But perhaps that just gives the wrong impression about the purpose of the document.

Reply:
I will concede that I skimmed over the second half of the abstract :) I saw this

>This document will be - at least at first - a proposal for how to organize data repositories in detail, focusing on collection naming conventions, filesystem locations, and developer workflows.

And did not fully appreciate the subsequent clarification that everything after assumes the "one big shared repository" model.

I still think the heavy focus on terminology is a lot to wade through, and the first part of Section 1 would be best used introducing the how and the why of the single shared repository model (leading naturally into collections within it).

Reply:
I've added another paragraph to the very first section, explaining why I'm advocating one big shared repo.